projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -734,13 +675,23 @@ export class IgxDateRangePickerComponent extends DisplayDensityBase
 
     /** @hidden @internal */
     public handleSelection(selectionData: Date[]): void {
+        const args: IBaseCancelableBrowserEventArgs = { owner: this, cancel: false };
+        this.selecting.emit(args);
```

Comment:
we could emit here the old and new value. This will allow user to validate the selection and to decide whether or not to cancel selection.